docs/doc-versions.adoc

```diff
@@ -3,9 +3,10 @@
 == Other documentation versions
 * https://docs.janusgraph.org/latest/index.html[Latest]
 * https://docs.janusgraph.org/0.3.0/index.html[Version 0.3.0]
+* https://docs.janusgraph.org/0.2.2/index.html[Version 0.2.2]
 * https://docs.janusgraph.org/0.2.1/index.html[Version 0.2.1]
 * https://docs.janusgraph.org/0.2.0/index.html[Version 0.2.0]
 * https://docs.janusgraph.org/0.1.1/index.html[Version 0.1.1]
 * https://docs.janusgraph.org/0.1.0/index.html[Version 0.1.0]
 
-This page only lists versions 0.3.0 and prior. Please visit the latest version to https://docs.janusgraph.org/latest/doc-versions.html[see a complete list of docs versions].
+Please visit the latest version to https://docs.janusgraph.org/latest/doc-versions.html[see a complete list of docs versions].
```

Comment:
Please visit the latest version **of this page** to see a complete list of docs versions.